sdk/python/kfp/compiler/_component_builder.py

```diff
@@ -374,18 +302,13 @@ def build_image_from_func(self, component_func, namespace, base_image, timeout,
 
       # Prepare build files
       logging.info('Generate build files.')
-      local_tarball_path = os.path.join(local_build_dir, 'docker.tmp.tar.gz')
-      self._prepare_buildfiles(local_tarball_path, local_docker_filepath, local_python_filepath, local_requirement_filepath)
-      self._build_image(local_tarball_path, namespace, timeout)
+      self._container_builder.build(local_build_dir, self._arc_docker_filename, self._target_image, timeout, namespace)
 
   def build_image_from_dockerfile(self, docker_filename, timeout, namespace):
```

Comment:
This function logically belongs to the container-building module.
We should probably deprecate it at some point.

Reply:
This is duplicate of the container_builder.build function. We can consider exposing the build function in the container_builder and deprecate this one.

Reply:
Is this function currently exposed?

Reply:
No. It is not. As discussed during the weekly meeting that it will not be exposed now.